backend/app/GraphQL/Queries/Questions.php

```diff
@@ -15,13 +15,13 @@ public function __invoke($_, array $args)
     {
         $query = Question::query();
         try {
-            if ($args['answered'] == 1) {
+            if (isset($args['answered']) && $args['answered']) {
                 $query->has('answers');
             } else {
                 $query->doesntHave('answers');
             }
 
-            if ($args['tag'] != null) {
+            if (isset($args['tag']) && $args['tag'] != null) {
```

Comment:
For line 18 and 24:
I think this is redundant? since `isset()` already checks if the variable is not null. Try it out lang.

Reply:
I've updated my PR based on this comment